www.foia.gov/_sass/layouts/_sidebar.scss

```diff
@@ -1,13 +1,22 @@
+$hide-tabs-width: 768px;
+
 .grid-flex {
   display: flex;
   justify-content: space-between;
 }
 .sidebar {
-  flex-basis: auto;
-  flex-grow: 5;
-  overflow: hidden; //the tabs are set to height: 100%, which overflows into the footer slightly
-  @media screen and (min-width: $medium-screen) {
-    min-width: 400px; // small as it can go without breaking tabs
+  display: none;
+  @media screen and (min-width: $hide-tabs-width) {
+    display: block;
+    flex-basis: auto;
+    flex-grow: 1;
+    overflow: hidden; //the tabs are set to height: 100%, which overflows into the footer slightly
```

Comment:
Wouldn't this be a bit neater if these rules were part of the default style (no media query) except for the `display:block`? If I understand the other breakpoints, they would override things correctly just as they are.